sopel/plugins/rules.py

```diff
@@ -1213,3 +1222,82 @@ def match_intent(self, intent):
         :rtype: bool
         """
         return bool(intent and self.INTENT_REGEX.match(intent))
+
+
+class FindRule(Rule):
+    """Anonymous find rule definition.
+
+    A find rule is like other anonymous rule with a twist: instead of maching
+    only once per IRC line, a find rule will execute for each non-overlapping
+    match for each of its regular expressions.
+
+    For example, to match for each word starting with the h letter in a line,
```

Comment:
Consistent with below.

```suggestion
    For example, to match for each word starting with the letter ``h`` in a line,
```